newIDE/app/src/AssetStore/PrivateAssets/PrivateAssetPackInformationPage.js

```diff
@@ -130,6 +140,121 @@ const PrivateAssetPackInformationPage = ({
       assetPack => assetPack.id === privateAssetPackListingData.id
     );
 
+  const packsIncludedInBundleTiles = React.useMemo(
```

Comment:
If i understand correctly, this is the list of bundles where this asset pack is included. And if the user owns one of the bundle, the card displays "you already own it". 
If I'm right, I think you should sort this list so that the bundle(s) that the user already owns are displayed first and not hidden by scroll.

Reply:
yep, I'll do that